### ISSUE TITLE
Error 'cannot forward the request' when running Tests

### ISSUE THREAD

**Issue by njeffers19** [zalando/zalenium#1074]
## 🐛 Bug Report

I have a Jenkins Server set up via a Docker Image. I am trying to get Zalenium to work with it. Jenkins is working well and I also was able to run tests
on it with an old Selenium server we have. So the tests themselves are working well.

## To Reproduce

Please see the Gist at https://gist.github.com/njeffers19/3b34c1b93cbb58c21bd9d49b41eeb7f1 for a detailed description of how to reproduce. This includes Docker Build/Run commands and the logs outputted from running the tests.

TLDR:> Running tests on Zalenium cause this error every single time. "The following error occurred: cannot forward the request unexpected end of stream on Connection"

## Expected behavior

Tests should run and open the webpage. Tests have been tested both locally and against a local selenium grid. 

## Environment

OS: Linux Redhat 8
Zalenium Image Version: Latest
Docker version: 1.13.1

**Comment by diemol** [comment]
Hi @njeffers19,

I apologize, it has taken a while for us to reply to open issues.

I am sorry, I will close this issue since we do not have enough information to reproduce it. The linked gist has a narrative description and we do not know where the used tests are. 

Feel free to open a new issue with a complete description.
